cloudimg/common.py

```diff
@@ -59,6 +59,29 @@ def object_name(self):
         return os.path.basename(self.image_path)
 
 
+class DeleteMetadata(object):
+    """
+    A collection of metadata necessary for deleting a disk image and it's
```

Comment:
```suggestion
    A collection of metadata necessary for deleting a disk image and its
```

`it's` is a contraction of "it is/has" only and not used for ownership.  The easiest way I have found to remember this is, whenever reading out a sentence with "it's", actually read it to yourself as "it is".  It makes mistakes stand out easily.

Reply:
yeah, I guess I had a bad moment when writing this :(